R/recipe.R

```diff
@@ -12,6 +12,13 @@ recipe <- function(x, ...) {
 #' @rdname recipe
 #' @export
 recipe.default <- function(x, ...) {
+
+  # Doing this here since it should work for all types of Matrix classes
```

Comment:
since it is a S4 class, I found this way of handling it the cleanest. It happens when you go `recipe(data, ...)`.